pkg/docker/schema/schema.json

```diff
@@ -109,6 +109,138 @@
             }
           },
           "additionalProperties": false
+        },
+        "push": {
+          "type": "object",
+          "properties": {
```

Comment:
In a follow-up PR, let's figure out which fields are required for each command and fill that in on the schema so that it can enforce that for us. For example, name is required for push.